### DISCUSSION TITLE
Adding Zalo as a messaging channel

### DISCUSSION THREAD

**Discussion by hiensarahly** [chatwoot/chatwoot#8457] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] --> Nope.

**Describe the solution you'd like**
A clear and concise description of what you want to happen. --> Integrate Zalo to the mix. Heavily used messaging app by individuals and businesses. Targeted in Vietnam.

**Describe alternatives you've considered**

**Additional context**

**Comment by iammattmartin** [comment]
Any docs you can provide?

**Comment by QuickSales** [comment]
> 

https://developers.zalo.me/docs

**Comment by Gum97** [comment]
up

**Comment by javier1234559** [comment]
Hi, I'm also interested in this feature, is anyone else building it?

**Comment by hsholding** [comment]
I have a Zalo Trigger node from n8n—how can I connect it into a Zalo chat channel in Chatwoot?

**Reply by diendh** [reply]
using API Channel.

**Comment by dongocanh0501** [comment]
please update it in the next version
